package.json

```diff
@@ -20,21 +20,19 @@
   },
   "dependencies": {
     "@antoinemopa/beamcoder": "^0.7.4",
-    "canvas": "^2.11.2",
+    "canvas": "^2.9.1",
```

Comment:
for framefusion version does not metter

Reply:
But it will match with luminary version, bumping it up in luminary could lead us to unexpcted problems, which I want to avoid at this time

Reply:
We need to bump in luminary soon to use node 20. So I would not downgrade here.